src/coreclr/nativeaot/Runtime/unix/UnixNativeCodeManager.cpp

```diff
@@ -1152,6 +1209,62 @@ int UnixNativeCodeManager::TrailingEpilogueInstructionsCount(MethodInfo * pMetho
         }
     }
 
+#elif defined(TARGET_RISCV64)
+
+// Load with immediate
+// LUI, LD, etc.
+// 0000 0000 0000 0000 1111 1111 1111 1111
+#define LUI_BITS 0x00000037
+#define LUI_MASK 0x0000007F
+
+// Load with register offset
+// LD with register offset
+// 0000 0000 0000 0000 0111 0000 0000 0000
+#define LD_BITS 0x00000003
+#define LD_MASK 0x0000007F
+
+// Branches, Jumps, System calls
+// BEQ, BNE, JAL, etc.
+// 1100 0000 0000 0000 0000 0000 0000 0000
+#define BEGS_BITS 0x00000063
+#define BEGS_MASK 0x7F000000
```

Comment:
1. BEGS_MASK doesn't line up with BEGS_BITS, it should be 0x0000007F like other masks (all 32-bit instruction opcodes sit in 7 lowest bits so all *_MASKs could be one variable).
2. This only checks for branches, opcodes for JALR, JAL, and SYSTEM are 0x67, 0x6f, 0x73 respectively